Updates/1090_dark_melodic_script.sql

```diff
@@ -0,0 +1,51 @@
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '3', '-129.399', '1005.35', '54.1277', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '4', '-129.399', '1005.35', '54.1277', '5.70723', '3000', '0', 'Dark Clearic Malod Change Orientation');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '5', '-129.399', '1005.35', '54.1277', '0.383972', '3000', '0', 'Dark Clearic Malod Change Orientation');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '6', '-138.249', '1017.98', '54.1666', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '7', '-133.805', '1024.5', '54.1383', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '8', '-133.805', '1024.5', '54.1383', '0.261799', '3000', '0', 'Dark Clearic Malod Change Orientation');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '9', '-133.805', '1024.5', '54.1383', '1.69297', '3000', '0', 'Dark Clearic Malod Change Orientation');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '10', '-133.805', '1024.5', '54.1383', '0.925025', '28000', '1943502', 'Dark Clearic Malod Second Event');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '11', '-157.823', '1010.55', '54.1879', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '12', '-172.394', '1017.44', '54.238', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '13', '-201.42', '1016.82', '54.3039', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '14', '-226.246', '1016.51', '54.3208', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '15', '-237.851', '1023.36', '54.309', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '16', '-234.792', '1026.51', '54.3085', '0', '34000', '1943503', 'Dark Clearic Malod third Event');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '17', '-201.884', '1016.94', '54.3022', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '18', '-166.062', '1017.46', '54.2065', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '19', '-151.175', '1012.26', '54.183', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '20', '-145.295', '1000.05', '54.2113', '0', '0', '0', 'Dark Clearic Malod Waypoint');
+INSERT INTO `creature_movement_template` VALUES ('19435', '0', '21', '-140.269', '982.351', '54.2018', '0', '0', '0', 'Dark Clearic Malod Waypoint');
```

Comment:
INSERT INTO `creature_movement_template` (`entry`, `pathId`, `point`, `position_x`, `position_y`, `position_z`, `orientation`, `waittime`, `script_id`, `comment`) VALUES
. 
.
. as kw mentioned add the table structure too just as safeguard incase there is table struct differences between classic-tbc-wotlk-db. Same for `dbscript_string` & `dbscripts_on_creature_movement`